mostlyai/__init__.py

```diff
@@ -1 +0,0 @@
-from .api import MostlyAI
```

Comment:
This might be an issue.

Perhaps we could change it to:
```python
from mostlyai.api import MostlyAI

__all__ = ['MostlyAI']
```
or equivalent to allow  `from mostlyai import MostlyAI`.